apps/docs/content/components/Pillar/index.mdx

```diff
@@ -54,7 +54,37 @@ If displaying an icon, do so for all cards to achieve a visually balanced layout
 
 When presenting more than four pillars, we recommend displaying them in multiple rows. You can use the [Grid](/components/Grid) or the [Stack](/components/Stack) component to achieve this. We recommend using a grid with 3 or 4 columns and balancing the number of pillars on each row.
 
-![An image showing the Pillar anatomy with an icon, a heading and a description in a vertical order.](https://github.com/primer/brand/assets/912236/1d031ae9-bf4e-4e19-a7b7-c8cc0b7e5940)
+![An image showing a group of numerous Pillars stacked in a 3 column grid.](https://github.com/primer/brand/assets/912236/1d031ae9-bf4e-4e19-a7b7-c8cc0b7e5940)
+
+### Pillar and Card
+
+Pillar and Card components might lead to ambiguity when deciding which one to use, as both could be used to provide pieces of related and condensed information. Some of the differences between them are:
+
+#### Interaction
+
+Pillars are used mainly for plain content and might include an optional link for extended information. On the other hand, Cards are primarily navigational elements.
+
+#### Hierarchy
+
+Cards should be used for noteworthy information to draw users attention. Pillars, in the other hand, should be used for common, less noticeable information compared to Cards.
+
+#### Group size
+
+Cards should be used in small groups (3, 4 or 5 items maximum) whereas Pillars might be used on both small and big groups (more than 5 items). The more items a users see's within a group, the less noticeable they become as unique elements.
```

Comment:
Would you ever want to use cards in a group of 5? 

```suggestion
Cards should be used in small groups (3 or 4 maximum), whereas Pillars can be used in larger or smaller groups (5+). The more items a users sees within a group, the less noticeable they become as unique elements.
```

Reply:
Or slight alternative to the last line..

```suggestion
Cards should be used in small groups (3 or 4 maximum), whereas Pillars can be used in larger or smaller groups (5+). As users encounter an increasing number of items within a group, the distinctiveness of each individual element diminishes.
```

Reply:
Love the last suggestion 👍